### PR TITLE
Changed doclinks, permissions and added samples for custodian and source collection

### PR DESCRIPTION
**Pull request by mahage-msft**
@bettirosengugi - This change contains documentation link changes that are part of https://github.com/microsoftgraph/microsoft-graph-docs/pull/11169.  This probably needs to be timed with that publication.  Laura is still working on final review, but hopefully we're really close.

### CODE REVIEW

File: sample-queries/sample-queries.json [hunk 1]
```diff
@@ -1965,8 +1965,8 @@
         "method": "GET",
         "humanName": "get ediscovery case",
         "requestUrl": "/beta/compliance/ediscovery/cases/{caseId}",
-        "docLink": "https://docs.microsoft.com/en-us/graph/api/ediscoverycase-get?view=graph-rest-beta",
-        "tip": "This query requires a case id. To find the id of a case; you can run: GET https://graph.microsoft.com/beta/compliance/ediscovery/cases",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/ediscovery-ediscoverycase-get?view=graph-rest-beta",
+        "tip": "Go to Modify Permissions tab and consent to eDiscovery.Read.All or eDiscovery.ReadWrite.All permission to run this query",
```
Comment (bettirosengugi) [inline sample-queries/sample-queries.json:1969]: It's still important to add this sentence that was there before: This query requires a case id. To find the id of a case; you can run: GET https://graph.microsoft.com/beta/compliance/ediscovery/cases to help users know how to get the id.

File: sample-queries/sample-queries.json [hunk 2]
```diff
@@ -1992,8 +1992,8 @@
         "method": "GET",
         "humanName": "list review sets",
         "requestUrl": "/beta/compliance/ediscovery/cases/{caseId}/reviewSets",
-        "docLink": "https://docs.microsoft.com/graph/api/reviewset-list?view=graph-rest-beta",
-        "tip": "Replace {caseId} with the ID from an eDiscovery Case (https://graph.microsoft.com/beta/compliance/ediscovery/cases).",
+        "docLink": "https://docs.microsoft.com/graph/api/ediscovery-reviewset-list?view=graph-rest-beta",
+        "tip": "Go to Modify Permissions tab and consent to eDiscovery.Read.All or eDiscovery.ReadWrite.All permission to run this query",
```
Comment (bettirosengugi) [inline sample-queries/sample-queries.json:1996]: This tip message would still be important to have: Replace {caseId} with the ID from an eDiscovery Case (https://graph.microsoft.com/beta/compliance/ediscovery/cases).

**Review by bettirosengugi — CHANGES_REQUESTED** [review]

File: sample-queries/sample-queries.json (hunk 1 above)
Comment (mahage-msft) [reply sample-queries/sample-queries.json:1969]: Fixed.

File: sample-queries/sample-queries.json (hunk 2 above)
Comment (mahage-msft) [reply sample-queries/sample-queries.json:1996]: fixed.

**Review by bettirosengugi — APPROVED** [review]

**Pull request merged**